.azure-pipelines/build-template.yml

```diff
@@ -0,0 +1,90 @@
+parameters:
+- name: cache_mode
+  type: string
+  values:
+  - wcache
+
+jobs:
+- job:
+  displayName: "broadcom"
```

Comment:
why cannot we trun each platform as a template as well?

Reply:
different platform has different target file name. It is hard to put into one template.
ex:
       broadcom has target of:  *.swi, *.raw, *.bin, *-dbg.bin.
        vs              has target of:  *.gz
  centec-arm64 has target of:  *.bin
      centec         has target of:  *-rpc.gz

Reply:
i think we can have several parameters saying on this platform whether we are building
- dbg images: true or false
- swi image: true or false
- raw images: true or false
- sync-rpc: true or false.

docker-sonic-vs.gz and docker-ptf can be special for vs image and they can be a separate build.